.github/workflows/preview.yml

```diff
@@ -96,12 +98,23 @@ jobs:
         run: |
           echo "The PR preview for ${{ fromJSON(env.PR_DATA).head_sha }} is available at [${{ env.PREVIEW_DOMAIN }}](https://${{ env.PREVIEW_DOMAIN }})" >> pr.md
           echo "" >> pr.md
-          echo "The following output files are affected by this PR:" >> pr.md
-          sed -E "s#(.*)#- [\1](https://${{ env.PREVIEW_DOMAIN }}/${{ fromJSON(env.PR_DATA).target_name }}/\1)#" diff.txt >> pr.md
-          echo "" >> pr.md
-          echo '[show diff](https://${{ env.PREVIEW_DOMAIN }}/diff.patch)' >> pr.md
-          echo "" >> pr.md
-          echo '[show diff as HTML](https://${{ env.PREVIEW_DOMAIN }}/diff.html)' >> pr.md
+
+          if [[ -f diff.txt ]] ; then
```

Comment:
```suggestion
          if [ -f diff.txt ] ; then
```
AFAIK, `[[ ]]` is for conditional command execution, whereas `[ ]` is `test`, which is probably what you're trying to do.
If you're using `[[ ]]` with an intent, can you please explain why is it better here than `[ ]` (`test`)?